codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttMessageBuilders.java

```diff
@@ -295,13 +296,18 @@ public MqttUnsubscribeMessage build() {
         }
     }
 
+    public interface PropertiesInitializer<T> {
+        void accept(T builder);
```

Comment:
@andsel one nit then we are ready to go... Make this method name something else. Maybe `init` or `apply` ?

Reply:
Done, `apply`